jbi/actions/default.py

```diff
@@ -25,17 +28,20 @@
 }
 
 
-def init(jira_project_key, **kwargs):
+def init(jira_project_key, sync_labels=True, **kwargs):
     """Function that takes required and optional params and returns a callable object"""
-    return DefaultExecutor(jira_project_key=jira_project_key, **kwargs)
+    return DefaultExecutor(
+        jira_project_key=jira_project_key, sync_labels=sync_labels, **kwargs
+    )
 
 
 class DefaultExecutor:
     """Callable class that encapsulates the default action."""
 
-    def __init__(self, jira_project_key, **kwargs):
+    def __init__(self, jira_project_key, sync_labels=True, **kwargs):
         """Initialize DefaultExecutor Object"""
         self.jira_project_key = jira_project_key
+        self.sync_labels = sync_labels
```

Comment:
Another option could be to omit the `sync_labels` params in `init` and `__init__` and instead do
```suggestion
        self.sync_labels = kwargs.get("sync_labels", True)
```

not strictly asking for a change here, just a suggestion.